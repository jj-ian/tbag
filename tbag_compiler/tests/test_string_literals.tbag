true {
	string a = "single quo'te";
	string newline = "\n";
	string c = "tab\ttab";
	string d = "backspac\be";
	string e = "carriage r\return";
	string f = "formfeed\fformfeed";
	string g = "escaped \"double quote\"";
	string h = "backslas\\h";
	
	print(a);
	print(newline);
	print(c);
	print(newline);
	print(d);
	print(newline);
	print(e);
	print(newline);
	print(f);
	print(newline);
	print(g);
	print(newline);
	print(h);
	print(newline);
	endgame;
}


