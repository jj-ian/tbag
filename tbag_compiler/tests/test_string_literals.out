single quo'te
tab	tab
backspace
carriage return
formfeedformfeed
escaped "double quote"
backslas\h
